server/auth_callout.go

```diff
@@ -116,73 +161,141 @@ func (s *Server) processClientOrLeafCallout(c *client, opts *Options) (authorize
 			// Operator mode is who we send the request on unless switching accounts.
 			issuer = acc.Name
 		}
-		// By default issuer needs to match server config or the requesting account in operator mode.
-		if arc.Issuer != issuer {
-			if !isOperatorMode {
-				respCh <- fmt.Sprintf("Wrong issuer for auth callout response on account %q, expected %q got %q", racc.Name, issuer, arc.Issuer)
-				return
-			} else if !acc.isAllowedAcount(arc.Issuer) {
-				respCh <- fmt.Sprintf("Account %q not permitted as valid account option for auth callout for account %q",
-					arc.Issuer, racc.Name)
-				return
-			}
+
+		// the jwt issuer can be a signing key
+		jwtIssuer := arc.Issuer
+		if !isOperatorMode && arc.IssuerAccount != "" {
```

Comment:
actually glad you like that - because I do as well - was just trying to match how DLC does it - but then there's always the ambiguity of `if false && false` which is not here...